pkg/mount/dag_uki_boot.go

```diff
@@ -0,0 +1,52 @@
+package mount
+
+import (
+	cnst "github.com/kairos-io/immucore/internal/constants"
+	"github.com/spectrocloud-labs/herd"
+)
+
+// RegisterUKI registers the dag for booting from UKI
+func (s *State) RegisterUKI(g *herd.Graph) error {
+	// Write sentinel
+	s.LogIfError(s.WriteSentinelDagStep(g), "sentinel")
+
+	s.LogIfError(s.LoadKernelModules(g), "kernel modules")
+
+	// Udev for devices discovery
+	s.LogIfError(s.UKIUdevDaemon(g), "udev")
```

Comment:
I wonder, here we need just for udev rules? or is just for having disk by label available ?

Reply:
Indeed, just for easy identification of disk by label. Could be moved into using whatever we want, but for ease of use and to reuse the steps from other paths, having the disks set up by label its the easieast way of finding them in the system